src/assets/js/src/mailoptin.js

```diff
@@ -698,9 +695,41 @@ var mailoptin_optin = {
         }
 
         this.show();
+
         $(this).trigger('moOptin:show', [optin_config.optin_uuid, optin_config, optin_type]);
     },
 
+    optin_sound_selected: function (optin_config, optin_type) {
+        //if we are in customizer, don't execute this to avoid double sound playing
+        if ($.MailOptin.is_customize_preview === true) return;
+
+        if(optin_type === 'bar' || optin_type === 'lightbox' || optin_type === 'slidein') {
+            let optin_sound_config = optin_config.optin_sound;
```

Comment:
Please replace every instance of let with var. We are still supporting older browser versions.